css/layout-css/news-feed-style.upload.css

```diff
@@ -412,6 +412,14 @@
   line-height: 1;
 }
 
+.new-news-feed-list-container .current-query {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: calc(100% - 185px);
```

Comment:
what's 185px?

Reply:
It's the space for the "Searching for:" label on the left, and a bit more (for buffer).

![image](https://user-images.githubusercontent.com/290733/59859220-408d4400-9374-11e9-9298-1e090dc88f0d.png)

Ideally, I'd change the HTML structure a bit to avoid needing to code in a number like this, but I'd like to get this fix QA tested for a deployment on Monday.